docs/install/remote.md

```diff
@@ -17,14 +17,14 @@ You can write your code on a local machine while running your functions on a rem
 Before you begin, ensure that the following prerequisites are met:
 
 1. Applications:
-   - Python 3.7   
+   - Python 3.7 or later 
```

Comment:
We don't support any python release, because this is dev, we can change it to 3.9

Reply:
Fixed